### PR TITLE
Alerting: Use RadioButtonList component to select alert condition

### LINKED ISSUE #48488: New alert condition picker
### Problem
In usability tests, users got confused between setting up a classic condition expression and picking an alert condition. It was hard for them to tell how alert conditions relate to queries and expressions. 

2022-02 Research Outcomes_ Alert creation (1)

### Solution
We are moving the alert condition closer to the queries and expressions and moving all options out of the dropdown as a radio button list.

2022-02 Research Outcomes_ Alert creation

### PR DESCRIPTION
**Pull request by konrad147**
**What this PR does / why we need it**:
This PR changes the alert condition picker

**Which issue(s) this PR fixes**:
Fixes #48488

**Special notes for your reviewer**:
**Preview**
image
image

### CODE REVIEW

File: public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx [hunk 1]
```diff
@@ -54,31 +54,32 @@ export const ConditionField: FC = () => {
...
+                required: {
+                  value: true,
+                  message: 'Please select the condition to alert on',
+                },
+              }}
+            />
+          </Field>
+        )}
+        {!options.length && (
+          <Alert title="Neither queries nor expressions have been configured" severity="warning">
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx:78]: I think it's probably easier without double negation :)

```suggestion
          <Alert title="No queries or expressions have been configured" severity="warning">
```

Comment (gillesdemey) [inline public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx:78]: The `Alert` inside the `Card` looks a bit weird, I'd move the warning outside of the card component

**Review by gillesdemey — APPROVED** [review]
Functionality is 👌 

Made some suggestions but feel free to merge if we're time constrained

File: public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx (hunk 1 above)
Comment (konrad147) [reply public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx:78]: I agree. The problem is that heading and margins come from the `Card` component

**Comment by peterholmberg** [comment]
@konrad147 can we make a follow up PR to have a max width on this component
image

**Pull request merged**